### PR TITLE
Fix: JS exports

### PR DESCRIPTION
**Pull request by gaebel**
Fixes JS exports

### CODE REVIEW

File: src/jsMain/kotlin/PayloadJS.kt [hunk 1]
```diff
@@ -9,3 +9,13 @@ data class SignUpResponseJS(
     val UserConfirmed: Boolean = false,
     val UserSub: String
 )
+
+data class HeaderKeyValuePair(
+    val key: String,
+    val `value`: String,
+)
```
Comment (gaebel) [inline src/jsMain/kotlin/PayloadJS.kt:16]:
```suggestion
```

Comment (gaebel) [reply src/jsMain/kotlin/PayloadJS.kt:16]: chore(js): remove unused code

File: src/jsMain/kotlin/PayloadJS.kt [hunk 2]
```diff
@@ -9,3 +9,8 @@ data class SignUpResponseJS(
     val UserConfirmed: Boolean = false,
     val UserSub: String
 )
+
+data class MfaSettingJS(
```
Comment (Ingwersaft) [inline src/jsMain/kotlin/PayloadJS.kt:13]: Why do you need JS classes for MfaSetting,  AssociateSoftwareTokenResponse and VerifySoftwareTokenResponse? They don't seem to be incompatible with js/ts, or am I missing something?

**Review by Ingwersaft — COMMENTED** [review]
Besides the one remark LGTM!

**Review by Ingwersaft — APPROVED** [review]
LGTM!

**Pull request merged**